apps/src/StudioApp.js

```diff
@@ -2019,12 +2017,6 @@ StudioApp.prototype.handleEditCode_ = function (config) {
     return;
   }
 
-  var displayMessage, examplePrograms, messageElement, onChange, startingText;
-
-  // Ensure global ace variable is the same as window.ace
-  // (important because they can be different in our test environment)
-  ace = window.ace;
```

Comment:
I don't understand under what circumstances this code actually does anything. `ace` isn't referenced anywhere in this file, and even if this line is considered to be setting the global `ace` variable, wouldn't this just be equivalent to `window.ace = window.ace`?